topics/statistics/tutorials/machinelearning/tutorial.md

```diff
@@ -128,14 +129,17 @@ The previous step produced a trained model (`zip` file) which we will use to pre
 
 
 # See predictions
-The last column of the predicted dataset shows the category of each row. A row either gets `0` (no breast cancer) or `1` (breast cancer) as its predicted category.
+The last column of the predicted dataset shows the category of each row. A row either gets `0` (no breast cancer) or `1` (breast cancer) as its predicted category. The first few rows are predicted as '0' or non-affected (no breast cancer).
```

Comment:
Maybe the first few rows are non-affected (no breast cancer). I am not sure if this information should be added as an observation.